decentraland/unity-renderer
# fix: Remove avatar loading animation if avatar appears inside modifier area

## What does this PR change?

Changes the visibility of the `AvatarWithHologram` animation depending if the player is teleported into an `AvatarHideModifierArea`. With this PR, the loading animation should not be seen.

It can happen that the animation is visible when starting. This is because there is no way to assure the order of the loading animation vs the triggering of entering the area. There is no way to predict it either. So, if the animation gets triggered, it should be turned off immediately.

## How to test the changes?

1) Open two instances of the race game: https://play.decentraland.zone/?renderer-branch=fix%2Fremove_avatar_particles_in_avatar_modifier&NETWORK=mainnet&position=-13%2C77&CATALYST=peer-testing.decentraland.org&realm=v2%7Epeer-testing.decentraland.org&DEBUG_SCENE_LOG=&island=Iv1n
2) Enter the race with one player. When the second player enters, no hologram or particle animation should be visible when moving the second player into the racing position. 

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
